nucypher/blockchain/eth/policies.py

```diff
@@ -23,9 +24,14 @@ def __init__(self, author: PolicyAuthor,
                  miner: Miner,
                  value: int,
                  lock_periods: int,
+                 expiration: maya.MayaDT,
                  *args, **kwargs):
         super().__init__(alice=author, ursula=miner, *args, **kwargs)
 
+        delta = expiration - maya.now()
```

Comment:
There is problem with this subtraction in windows: result is seconds but treated as days

Reply:
Hmm, should be of timedelta type, I guess

Reply:
The datetime/period conversion code here is also included in a utility function in `constants.py` with fixes for windows.

Reply:
Yeah, I mean I just moved this from one place to another obviously.  I see the convenience method that y'all have pointed out; we'll use that in the future.